### PR TITLE
Change alias method chain to equivalent alias_method

### PR DESCRIPTION
**Pull request by tuned-up**
This will add support for latest Rails.

### CODE REVIEW

**Comment by tuned-up** [comment]
it fails since rails5 requires ruby >= 2.2.2.
Is it even possible to use different scripts for different ruby versions?

**Comment by tuned-up** [comment]
Travis fails since rails5 requires ruby >= 2.2.2.
I tried to change travis.yml is a way that it only runs for valid ruby\gemfile pairs, but now it skips bundle. Do you have any ideas on how to fix it?

**Comment by tuned-up** [comment]
Travis fails since rails5 requires ruby >= 2.2.2.
Since the only way to run custom gemfiles against old ruby (like, 1.9.3 with rails3 and not 1.9.3 with rails5) is through custom shell script, i stripped old rubies and rails from .travis.yml.
Please tell me what you think

**Comment by Napolskih** [comment]
Hi
Thanks for the feedback. I will answer later until I'm busy. Use while fork please.

File: Appraisals [hunk 1]
```diff
@@ -1,15 +1,13 @@
-appraise "rails3.0" do
-  gem "activesupport", "~> 3.0.0"
+appraise "rails4.0" do
+  gem "activesupport", "~> 4.0.0"
 end
```
Comment (Napolskih) [inline Appraisals]: add please 3.2/4.1/4.2

**Comment by Napolskih** [comment]
maybe drop 1.9.3 support and use prepend ?

File: Appraisals (hunk 1 above)
Comment (tuned-up) [reply Appraisals]: done in `036ffdd`

File: Appraisals [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
-appraise "rails3.1" do
-  gem "activesupport", "~> 3.1.0"
+appraise "rails4.0" do
+  gem "activesupport", "~> 4.0.0"
 end
 
-appraise "rails3.2" do
-  gem "activesupport", "~> 3.2.0"
+appraise "rails4.1" do
+  gem "activesupport", "~> 4.0.0"
```
Comment (Napolskih) [inline Appraisals]: 4.1

File: Appraisals [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
 end
 
-appraise "rails4.0" do
+appraise "rails4.2" do
   gem "activesupport", "~> 4.0.0"
```
Comment (Napolskih) [inline Appraisals]: 4.2

Comment (tuned-up) [reply Appraisals]: missed it.
Fixed.

File: Appraisals [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
 
 appraise "rails4.0" do
   gem "activesupport", "~> 4.0.0"
-end
\ No newline at end of file
+end
+
+appraise "rails4.1" do
+  gem "activesupport", "~> 4.1.0"
+  gem "minitest", '~> 5.0'
```
Comment (tuned-up) [inline Appraisals]: activesupport 4.1 requies minitest ~> 5.1

File: Appraisals [hunk 5]
```diff
@@ -1,15 +1,27 @@
...
 
 appraise "rails4.0" do
   gem "activesupport", "~> 4.0.0"
-end
\ No newline at end of file
+end
+
+appraise "rails4.1" do
+  gem "activesupport", "~> 4.1.0"
+  gem "minitest", '~> 5.1'
```
Comment (tuned-up) [inline Appraisals]: activesupport ~> 4.1 requies minitest ~> 5.1

**Pull request merged**